### ISSUE TITLE
Missing audio output in the desktop version of Telegram

### ISSUE THREAD

**Issue by dimon2070** [kcat/openal-soft#790]
Hi all, I'm missing audio output in the desktop version of Telegram. Please help me to solve the problem!

OS: Win 10.
Audio Interfaces: , M-Audio M-Track Solo (last driver version 2021), M-Audio MobilePre USB (last driver version 2009).

1. Audio and voice messages are not playing - stop at the 00:00 position.
2. Sometimes there is audio, but it plays with a gate effect, that is, there are pauses in the audio stream, as if the download is slowing down.
3. Audio from video plays until the first/second timeline rewind, then disappears afterwards.

 What was done:
1. Reinstalling the application.
2. Reinstalling audio driver.
3. Disconnecting and uninstalling all other audio drivers not related to the audio card used.
4. Reinstalling the OS.
5. I even installed an older version of the OS - Win7, but nothing helped😅

Thanks!
[last_openal_log.txt](https://github.com/kcat/openal-soft/files/10179909/last_openal_log.txt)

**Comment by kcat** [comment]
I presume you got the issue fixed since you closed it? How did you fix it, and what was the problem?

**Comment by dimon2070** [comment]
> I presume you got the issue fixed since you closed it? How did you fix it, and what was the problem?

Problem
The problem was that when selecting the driver of the appropriate device (MobilePre), the Telegram app still left the driver of another device active (M-Track Solo, before that driver was uninstalled through the standard Windows uninstaller). In the settings window, despite this, the correct driver was displayed. I noticed a discrepancy in the last log.

Solution
0. Uninstalled Telegram.
1. I used the Disk Cleanup tool to remove the deprecated drivers (by checking the box). 
2. I cleaned the registry to find the name of the driver I did not want to use.
3. After that, I deleted all audio devices in Device Manager.
4. Installed the necessary driver.
5. Installed Telegram.
6. Profit
